platforms/maven/pom.xml

```diff
@@ -0,0 +1,247 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>opencv.org</groupId>
+    <artifactId>opencv</artifactId>
+    <packaging>bundle</packaging>
+    <version>3.1.0</version>
```

Comment:
There appears to be no Maven way to set the version into the POM at build time; a hard-coded value is unavoidable. However I have made the addition of a further 'enforcer' rule to ensure the POM version matches that extracted from the header file. A mis-match causes the build to fail.